app/views/projects/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h3> Project Information</h3>
+<h3> Name: <%= @project.name %></h3>
+<h3> Material: <%= @project.material %></h3>
+<h3> Challenge Theme: <%= @challenge.theme %></h3>
+<h3> Number of Contestants: <%= @contestants_count %></h3>
```

Comment:
See my comment in your `projects` controller. We would just want to reach through `@project` to get to its `challenges` to get to a challenges theme. 
```ruby
@project.challenges.theme
```
On line 5, see my comment in your `projects` controller but you would create a method in your `Project` model that can get the count of contestants. Then call that method on your `@project`.